### ISSUE TITLE
Question regarding Freihand Bounding boxes

### ISSUE THREAD

**Issue by pablovela5620** [mks0601/I2L-MeshNet_RELEASE#17]
I see that you got the bounding box info from Detectron2, but what training data did you use to create the hand detector? Did you just use the smallest rectangle that encompassed all the projected 2D keypoints? Did you use a pretrained hand detector on some other data? 

Thanks!

**Comment by mks0601** [comment]
Q. Did you just use the smallest rectangle that encompassed all the projected 2D keypoints?
A. Yes!

I trained a Mask R-CNN ResNet50-FPN model (without segmentation estimation) to detect bbox of hands. The GT hand bbox are created by extending 50% of projected 2D keypoints.

**Comment by pablovela5620** [comment]
Exactly what I was looking for, thank you for being so quick to respond!
